### ISSUE TITLE
Issue when pasting in the same address with a leading "oeth:"

### ISSUE THREAD

**Issue by ZakGriffith** [BuidlGuidl/address-vision-port#56]
This is very much not critical and 99% of people won't run into it.  If it is a non-issue, just close this.

  When I already have an address loaded.  If the same address is pasted in with the leading oeth: text, it doesn't fully reload the address.  

![oEthVision](https://github.com/BuidlGuidl/address-vision-port/assets/12072395/d781e301-60ea-4c80-9cd0-06a2fa86bb0b)

**Comment by portdeveloper** [comment]
The reasoning behind this behavior is the following:
Since we have two sources of truth: 
1. The url bar
2. The search bar
and they keep each other updated, I needed a way to check if the input and the url are the same and if they are, the app does not update anything, since it is the same address. 

But let me look for a solution so that it gets updated! 
Perhaps I need to rewrite the key logic of the app...

**Comment by ZakGriffith** [comment]
I don't believe that should be necessary for such a little that that people shouldn't run into.  Don't spend much time on it.

**Comment by ZakGriffith** [comment]
Closing as fixing this would introduce more issues, and this will not likely happen to anyone.
